sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -789,6 +847,16 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
 
   struct PathAppendDataResult
   {
+    Azure::Core::Nullable<std::string> ETag;
```

Comment:
Why nullable?

Reply:
Talked offline, server behavior.

Reply:
ditto

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/1637f4b9ed998c40d9ae3fe054de3382b4769150/sdk/storage/Azure.Storage.Files.DataLake/src/Generated/DataLakeRestClient.cs#L3656

ditto.